src/tools/ScenarioMeasurement/SizeOnDisk/SizeOnDisk.cs

```diff
@@ -0,0 +1,130 @@
+                counters.Add(new Counter { MetricName = "count", TopCounter = directoryIsTop, Name = $"{RemoveVersions(directory.Key)}", Results = new[] { (double)resultCount } });
+                foreach (var file in directory.Value)
+                {
+                    counters.Add(new Counter { MetricName = "bytes", Name = $"{RemoveVersions(file.Key)}", Results = new[] { (double)file.Value } });
+                }
+            }
+            counters.Add(new Counter { MetricName = "bytes", Name = scenarioName, DefaultCounter = true, TopCounter = true, Results = new[] { (double)totalSize } });
+            counters.Add(new Counter { MetricName = "count", Name = scenarioName, TopCounter = true, Results = new[] { (double)totalCount } });
+            var reporter = Reporter.CreateReporter();
+            if (reporter != null)
+            {
+                var test = new Test();
+                test.Categories.Add("SizeOnDisk");
+                test.Name = scenarioName;
+                test.AddCounter(counters);
+                reporter.AddTest(test);
+                if (!String.IsNullOrEmpty(reportJsonPath))
+                {
+                    var json = reporter.GetJson();
+                    if(json != null)
+                    {
+                        File.WriteAllText(reportJsonPath, json);
+                    }
+                }
+            }
+            Console.WriteLine(reporter.WriteResultTable());
+            return 0;
+        }
+
+        static HashSet<string> versions = new HashSet<string>();
+
+        static string RemoveVersions(string name)
+        {
+            foreach(var version in versions)
+            {
+                name = name.Replace(version, "VERSION");
+            }
+            return name;
+        }
+
+        static void FindVersionNumbers(string path)
+        {
+            var sdkDir = Path.Combine(path, "sdk");
+            if (!Directory.Exists(sdkDir))
+                return;
+            var sharedDir = Path.Combine(path, "shared");
+            var sdkVersion =  new DirectoryInfo(Directory.GetDirectories(sdkDir).Single()).Name;
+            var templateDir = Path.Combine(path, "templates");
+
+            versions.Add(sdkVersion);
+            // the Templates dir seems to have the same version as the SDK version, but with a slightly different format
+            versions.Add(Regex.Replace(sdkVersion, @"(\d+\.\d+\.\d)00", "$1"));
+
+            versions.Add(new DirectoryInfo(Directory.GetDirectories(templateDir).Single()).Name);
+            
+            foreach(var dir in Directory.GetDirectories(sharedDir))
+            {
+                versions.Add(new DirectoryInfo(Directory.GetDirectories(dir).Single()).Name);
+            }
+            foreach (var version in versions) Console.WriteLine(version);
```

Comment:
nit: Braces

Reply:
Good catch. This was actually just debug code I forgot to remove.